### PR TITLE
fix community category divider label margin

### LINKED ISSUE #19204: Communities - Community page issues when scrolling
# Design Feedback 2.27

## [Communities] Issues when scrolling a community page screen
Figma frame of the Review [here](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj/Release-Reviews?type=design&node-id=4%3A111403&mode=design&t=IwKFMlZGTBic5nwK-1)

- [x] - **Status bar isn't visible** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35475&mode=design#725950910)] might be related to this issue logged here: status-im/status-mobile/issues/15260

- [ ] - **Channel top bar blur** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=4:111403&mode=design#727200564)] isn't behaving correctly causing the community header image to not be visible

- [x] - **Top bar gets cut off** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35475&mode=design#725946599)] when scrolling (see the narrow margin between ❌ button and top nav bottom)

- [ ] - **Divider label margin** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35475&mode=design#725948018)] of channel categories doesn't match design

Screenshot 2024-03-12 alle 14 56 08

### Reproduction

1. Open a community
2. Scroll channel list

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS
- Devices in the screenshots: iPhone 15 Pro Max

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/19204 (issue 4)

### Testing Note:

Skipping manual qa as PR fixes small design related issue and manually tested.

status: ready

### CODE REVIEW

File: src/status_im/contexts/communities/overview/view.cljs [hunk 1]
```diff
@@ -78,16 +79,17 @@
...
       (when-not collapsed?
         [rn/view {:style {:padding-horizontal 8}}
-         (for [chat chats]
-           ^{:key (:id chat)}
-           [channel-chat-item community-id chat])])])])
+         (let [chats-count (count chats)]
+           (for [index (range chats-count)]
+             (let [chat (get chats index)]
+               ^{:key (:id chat)}
+               [channel-chat-item community-id chat (= index (dec chats-count))])))])])])
```
Comment (ibrkhalil) [inline src/status_im/contexts/communities/overview/view.cljs]: nit: I'd assume that using `map-indexed` might be faster, But I am not sure

**Review by ibrkhalil — APPROVED** [review]

**Comment by Parveshdhull** [comment]
@Francesca-G PR is ready for design review, thank you

**Comment by mariia-skrypnyk** [comment]
Hi @Parveshdhull !
Thanks for your PR!

Please, next time add description why this PR should skip manual testing as it described here: https://github.com/status-im/status-mobile/blob/develop/doc/pipeline_process.md

It will help QA's not to spend so much time looking into linked issues/comment etc to analize failed e2e and potential issues.
It would be super helpful.
Thanks! 🙏

Failed 2e2 are not related and I'll move PR to @Francesca-G review.

**Review by Francesca-G — APPROVED** [review]
👍✨

**Pull request merged**